src/ui/states/state-needs-payment-info.svelte

```diff
@@ -75,6 +76,11 @@
       onContinue();
     }
   };
+
+  const theme = new Theme(brandingInfo?.appearance);
+
+  let shapeCustomisation = theme.shape;
```

Comment:
Stripe elements to not support css variables, using directly the css rules calculated by the Theme.